.github/workflows/rumble-vulnerability-data.yaml

```diff
@@ -4,44 +4,69 @@ on:
     - cron: "1 5 * * *"
   workflow_dispatch:
   push:
-    branches: [rumble-vulnerability-data]
+    branches: [rumble-insights]
+
+env:
+  PROJECT_ID: "${{ secrets.PROJECT_ID }}"
+  WORKLOAD_IDENTITY_PROVIDER: "${{ secrets.GCP_WORKLOAD_IDENTITY_PROVIDER }}"
+  SERVICE_ACCOUNT: "${{ secrets.GH_ACTION_SERVICE_ACCOUNT }}"
+  GH_TOKEN: ${{ github.token }}
 
 defaults:
   run:
     shell: bash
     working-directory: ./tools/rumble
 
 jobs:
-  generate-vulnerability-json:
+  generate-csvs-and-json:
     runs-on: ubuntu-latest
 
     permissions:
-      id-token: write # federate with GCP
-
+      contents: read
```

Comment:
Can we retain and document the permissions in the workflows?

Reply:
Ah yeah, no idea where that got lost in the shuffle, will add it back for sure